crates/stats-agg/src/stats1d.rs

```diff
@@ -364,11 +415,82 @@ mod tests {
         assert_relative_eq!(p.sx4, 1000.8186787745212);
     }
 
+    #[test]
+    fn test_against_known_vals_tf() {
+        let p = StatsSummary1D::new_from_vec(vec![tf(7.0), tf(18.0), tf(-2.0), tf(5.0), tf(3.0)])
+            .unwrap();
+
+        assert_eq!(p.n, 5);
+        assert_relative_eq!(Into::<f64>::into(p.sx), 31.);
+        assert_relative_eq!(Into::<f64>::into(p.sx2), 218.8);
+        assert_relative_eq!(Into::<f64>::into(p.sx3), 1057.68);
+        assert_relative_eq!(Into::<f64>::into(p.sx4), 24016.336);
+
+        let p = p.remove(tf(18.0)).unwrap();
+
+        assert_eq!(p.n, 4);
+        assert_relative_eq!(Into::<f64>::into(p.sx), 13.);
+        assert_relative_eq!(Into::<f64>::into(p.sx2), 44.75, epsilon = 0.0000001);
```

Comment:
Why does this need to change the epsilon, it may be necessary, but I'd love to have a note as to why this needs it if it does.

Reply:
I added explanations for all of them.